internal/auth/repository/eventsourcing/handler/user_session.go

```diff
@@ -48,6 +48,8 @@ func (u *UserSession) Reduce(event *models.Event) (err error) {
 		es_model.HumanExternalLoginCheckSucceeded,
 		es_model.HumanMFAOTPCheckSucceeded,
 		es_model.HumanMFAOTPCheckFailed,
+		es_model.HumanMFAU2FTokenCheckSucceeded,
+		es_model.HumanMFAU2FTokenCheckFailed,
```

Comment:
Is passwordless not relevant here?
I think remove u2f, remove passwordless should alsow be handles in user session

Reply:
added passwordless and remove are now handled on all of the user sessions as well